File: FizzBuzz.cpp
#include <iostream>
using namespace std;

void FizzBuzz(int);

int main() {
	FizzBuzz(100);
	return 0;
}

void FizzBuzz(int i) {
	if(i==0) return;
	else {
		FizzBuzz(i-1);
		if(i%3==0&&i%5==0) cout<<"FizzBuzz"<<endl;
		else if(i%3==0) cout<<"Fizz"<<endl;
		else if(i%5==0) cout<<"Buzz"<<endl;
		else cout<<i<<endl;
	}
}

File: FizzBuzz.py
import os

def FizzBuzz(i):
    if(i==0):
        return
    else:
        FizzBuzz(i-1)
        if(i%3==0 and i%5==0):
            print("FizzBuzz")
        elif(i%3==0):
            print("Fizz")
        elif(i%5==0):
            print("Buzz")
        else:
            print(i)
 
FizzBuzz(100) 
    
os.system("pause")
